packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card.tsx

```diff
@@ -366,7 +421,13 @@ export const PropertyTypeRow = ({
   });
 
   const propertyTypes = usePropertyTypes();
-  const property = propertyTypes?.[$id];
+  const { propertyTypes: entityTypePropertyTypes } = useEntityType();
+  const property = entityTypePropertyTypes[$id] ?? propertyTypes?.[$id];
+
+  const [currentVersion, latestVersion] = usePropertyTypeVersions(
+    $id,
+    propertyTypes,
+  );
```

Comment:
These would always be the same if it isn't already on the entity, right?

Reply:
If the property type is being added you mean?

Reply:
Yes – if this property type only exists in `propertyTypes` then presumably we already have the latest (unless `propertyTypes` can end up with multiple versions of the same one). Not a big deal though

Reply:
There can be multiple versions of the same one inside `propertyTypes`, by updating a property type (the previous and new version get stored there). In the end I don't think it's worth doing a lot optimizations here since we won't be fetching all the latest property types in the future anyway, right?

Reply:
Sure, I don't think it's worth it